resources/public/api/conf/4.0/loss_claims_create.yaml

```diff
@@ -1,147 +1,148 @@
+          </tr>
+      </tbody>
+      <tbody>
+          <tr>
+              <td><p>NOT_FOUND</p></td>
+              <td><p>Simulates the scenario where no data is found.</p></td>
+          </tr>
+      </tbody>
+      <tbody>
+          <tr>
+              <td><p>DUPLICATE</p></td>
+              <td><p>Simulates the scenario where a duplicated value is found.</p></td>
+          </tr>
+      </tbody>
+      <tbody>
+          <tr>
+              <td><p>ACCOUNTING_PERIOD_NOT_ENDED</p></td>
+              <td><p>Simulates the scenario where the Loss claim has been made too soon.</p></td>
+          </tr>
+      </tbody>
+      <tbody>
+          <tr>
+              <td><p>NO_ACCOUNTING_PERIOD</p></td>
+              <td><p>Simulates the scenario where there is no accounting period for the year of the claim.</p></td>
+          </tr>
+      </tbody>
+    </table>
 
-tags:
-  - Loss Claims
-  
-parameters:
-  - $ref: './common/pathParameters.yaml#/components/parameters/nino'
-  - $ref: './common/headers.yaml#/components/parameters/acceptHeader'
-  - $ref: './common/headers.yaml#/components/parameters/contentTypeJson'
-  - $ref: './common/headers.yaml#/components/parameters/authorizationWriteScopeHeader'
-  - $ref: './common/headers.yaml#/components/parameters/testScenarioHeader'
+  tags:
+    - Loss Claims
 
-security:
-  - User-Restricted:
-      - write:self-assessment
+  parameters:
+    - $ref: './common/pathParameters.yaml#/components/parameters/nino'
+    - $ref: './common/headers.yaml#/components/parameters/acceptHeader'
+    - $ref: './common/headers.yaml#/components/parameters/contentTypeJson'
+    - $ref: './common/headers.yaml#/components/parameters/authorizationWriteScopeHeader'
+    - $ref: './common/headers.yaml#/components/parameters/testScenarioHeader'
 
-requestBody:
-  content:
-    application/json:
-      schema:
-        $ref: './schemas/lossClaims/createLossClaim/create_request.json'
-      examples:
-        example-1:
-          description: Self-Employment Claim Request
-          value:
-            $ref: './examples/lossClaims/createLossClaim/create_request_self_employment_claims.json'
-        example-2:
-          description: UK Property Claim Request
-          value:
-            $ref: './examples/lossClaims/createLossClaim/create_request_property_claims.json'
-        example-3:
-          description: Foreign Property Claim Request
-          value:
-            $ref: './examples/lossClaims/createLossClaim/create_request_foreign_property_claims.json'
+  security:
+    - User-Restricted:
+        - write:self-assessment
 
-responses:
-  "201":
-    description: Success
-    headers:
-      X-CorrelationId:
-        $ref: './common/headers.yaml#/components/parameters/correlationId'
+  requestBody:
     content:
       application/json:
         schema:
-          $ref: './schemas/lossClaims/createLossClaim/create_response.json'
+          $ref: './schemas/lossClaims/createLossClaim/create_request.json'
```

Comment:
This is a very misleading diff. The only change to this file is all the lines are indented by one and a new `post:` line was added to the top.